File: PHP/ReverseString.php
<?php
/**
 * @version PHP 8.2.0
 * @author MARIO SAM <eu@mariosam.com.br>
 * @see I would love to work with you instead solving web code tests: hire me!
 */
namespace PHP;

class ReverseString {    

    /**
     * @param String[] $s
     * @return NULL
     */
    function reverseString($s) {
        $i = 0;
        $j = count($s) - 1;
        while ($i < count($s) / 2) {
            $temp = $s[$i];
            $s[$i] = $s[$j];
            $s[$j] = $temp;
            $i++;
            $j--;
        }
    }

}

File: JAVA/ReverseString.java
/**
 * @version JAVA
 * @author MARIO SAM <eu@mariosam.com.br>
 * @see I would love to work with you instead solving web code tests: hire me!
 */
package JAVA;

public class ReverseString {

    public static void main(String[] args) {
        System.out.printf("Resultado: %d\n", reverseString( new char[]{'a'} ));
    }

    public static Object[] reverseString(char[] s) {
        for (int i = 0, j = s.length - 1; i < s.length / 2; i++, j--) {
            char temp = s[i];
            s[i] = s[j];
            s[j] = temp;
        }
        return null; 
    }

}
